src/utils/hub.js

```diff
@@ -407,6 +407,7 @@ export async function getModelFile(path_or_repo_id, filename, fatal = true, opti
             throw Error("Invalid configuration detected: both local and remote models are disabled. Fix by setting `env.allowLocalModels` or `env.allowRemoteModels` to `true`.")
         }
     }
+    return_path &&= env.useFSCache || env.useCustomCache;
```

Comment:
Just need to make sure this doesn't break some use-case when `useCustomCache=true`